docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
+
+.. note:: For a detailed introduction to set up Salesforce CLI and Visual Studio Code to work with CumulusCI, we recommend completing the `Build Applications with CumulusCI <https://trailhead.salesforce.com/en/content/learn/trails/build-applications-with-cumulusci>`_ module on Trailhead.
 
-Predefined orgs
+To set up Salesforce DX:
+
+1. `Install Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.228.0.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ 
+
+.. important :: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your devhub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
+
+.. note:: SFDX supports multiple DevHubs, so CumulusCI uses the one set as ``defaultdevhubusername`` when creating scratch orgs.
+
+Learn more about Salesforce DX at https://developer.salesforce.com/platform/dx.
+
+
+
+Predefined Orgs
 ---------------
 
-Create a scratch org
+CumulusCI comes with predefined org configurations. Every org's keychain starts with these configurations ready and available to be turned into a live scratch org. To see the predefined org configuations in your project:
+
+.. code-block:: console
+
+    $ cci org list
+
+
++-------------+--------------------------+-----------------------+----------+
+|   Org       | Role                     | Configuration File    | Lifespan |
++=============+==========================+=======================+==========+
+| ``dev``     | Development workflows    | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``qa``      | Testing workflows        | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``feature`` | Continuous Integration   | ``orgs/dev.json``     | 1 day    |
++-------------+--------------------------+-----------------------+----------+
+| ``beta``    | | Continuous Integration | ``orgs/beta.json``    | 1 day    |
+|             | | Hands-On Testing       |                       |          |
++-------------+--------------------------+-----------------------+----------+
+| ``release`` | | Continuous Integration | ``orgs/release.json`` | 1 day    |
+|             | | Hands-On Testing       |                       |          |
+|             | | Demo Workflow          |                       |          |
++-------------+--------------------------+-----------------------+----------+
```

Comment:
The bullet list version of the orgs seemed to be a bit repetitive in its structure, so I thought a table would be even better. (I left the bullet list just below for comparison. Let me know which one you prefer.)

Reply:
I really like how clean this table is. Unfortunately this branch isn't using the updated theme that the final version will have. My vote is still for the table though. I think we should move the command to list the orgs in a project below the table (see edits above and below).